charts/tempo-distributed/values.yaml

```
@@ -1914,7 +1916,7 @@ minio:
 # Configuration for the gateway
 gateway:
   # -- Specifies whether the gateway should be enabled
-  enabled: false
+  enabled: true
```

it should be false by default
```suggestion
  enabled: false
```